net/spoofer/Makefile

```diff
@@ -0,0 +1,79 @@
+#
+# Copyright (C) 2018 The Regents of the University of California
```

Comment:
do you _really_ have copyright assignment on file with UC? You can't just make this up.  Either drop it or put your own name here, or at least confirm that you really do have assignment rights....

Reply:
Yes, I do.  I'm the same person who releases the upstream spoofer package.

Reply:
No problem, it's something that's often done wrongly here, so it's just something I'll mention to have people check :)